src/test/resources/io/vertx/openapi/contract/split/malformedComponents.json

```diff
@@ -0,0 +1,24 @@
+{
+  "title": "my json api",
+  "description": "taken from https://stackoverflow.com/questions/16826128/why-is-this-json-schema-invalid-using-any-type",
```

Comment:
Do you know the license of the Stackoverflow example? If not we should create our own to avoid legal issues.

Reply:
i've removed this example and replaced with my own.